src/CategoryCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardHeader, IconButton, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(1),
+    maxWidth: 500,
+    margin: '10px auto',
+  },
+}));
+
+type Props = {
+  title: string;
+  description: string;
+  avatarSrc: string;
+};
+
+export const CategoryCard: React.FC<Props> = ({
+  title,
+  description,
+  avatarSrc,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <div className="CategoryCard">
```

Comment:
Is this div necessary? I don't see any change after removing it, so maybe it's just some leftover?

Reply:
Done ✅